docs/funding.qmd

```diff
@@ -0,0 +1,23 @@
+---
+title: "Funding"
+---
+
```

Comment:
Maybe it's worth adding a small `UU Credits` section before the `Small Compute application` section, reiterating that we can provide an initial starting budget. Otherwise users who land on this page might be unnecessarily put off by the idea of having to do an application.